### PR TITLE
Allow setting the strict parameter when initializing Evaluation

### PR DESCRIPTION
**Pull request by mexicat**
Needed for non-strict evaluation from the server.

### CODE REVIEW

File: konfuzio_sdk/evaluate.py [hunk 1]
```diff
@@ -217,14 +217,15 @@ class Evaluation:
 
     from konfuzio_sdk.data import Document
 
-    def __init__(self, documents: List[Tuple[Document, Document]]):
+    def __init__(self, documents: List[Tuple[Document, Document]], strict: bool = True):
         """
         Relate to the two document instances.
 
         :param documents: A list of tuple Documents that should be compared.
+        :param strict: A boolean passed to the `compare` function.
```
Comment (atraining) [inline konfuzio_sdk/evaluate.py:225]: `By default the evaluation is strict. A strict evaluation compares the exact start and end offset of a Span. If the evaluation is not strict the predicted text just needs to be within the annotated text by the human.`

File: konfuzio_sdk/evaluate.py [hunk 2]
```diff
@@ -217,14 +217,15 @@ class Evaluation:
...
         """
         self.documents = documents
-        self.strict = True
```
Comment (atraining) [inline konfuzio_sdk/evaluate.py:227]: Please add one test for strict = False using the `__init__`.

File: konfuzio_sdk/evaluate.py (hunk 1 above)
Comment (mexicat) [reply konfuzio_sdk/evaluate.py:225]: Do I need to add this text to the docstring? I think this should be explained in the `compare` function instead.

File: konfuzio_sdk/evaluate.py (hunk 2 above)
Comment (mexicat) [reply konfuzio_sdk/evaluate.py:227]: Done.

**Pull request merged**